css/components/service-cta-block.css

```diff
@@ -1,7 +1,7 @@
 .service-cta-block__list {
   padding-left: 0;
   list-style: none;
-  border-bottom: var(--border);
+  border-bottom: var(--service-cta-border);
```

Comment:
This is fine, but 'only' if `--service-cta-border` evaluates to `--border` (and then it also won't break existing sites.